sdk/storage/storage-blob/src/credentials/SharedKeyCredential.browser.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export class SharedKeyCredential {}
```

Comment:
this file could be made simpler/smaller: `export const SharedKeyCredential = undefined`.

Reply:
As you wished!